restraint/core.py

```diff
@@ -60,7 +62,10 @@ def perms(self):
     def _load_perms(self, account, which_perms):
         perm_set_names = self._config['perm_set_getter'](account)
         perm_levels = models.PermLevel.objects.filter(
-            permaccess__perm_set__name__in=perm_set_names).select_related('perm')
+            Q(permaccess__perm_set__name__in=perm_set_names) | Q(
+                permaccess__perm_user_id=account.id,
+                permaccess__perm_user_type__app_label=account._meta.app_label,
```

Comment:
Why is this not a ContentType query? Is this more performant? Might this be subject to breakage?